ci/scenarios/vPool/add_remove_vpool_concurrently/main.py

```diff
@@ -73,36 +71,39 @@ def add_vpools(self):
                                                    'storage_ip': sr.ip,
                                                    'storagerouter_ip': sr.ip,
                                                    'vpool_name': vpool_name,
-                                                   'writecache_size': 10}})
+                                                   'writecache_size': 5}})
             tasks[vpool_name] = self.api.post(api='storagerouters/{0}/add_vpool'.format(sr.guid), data=data)
-
         for vpool_name, task in tasks.iteritems():
             self.LOGGER.info('-> {0} running: {1}'.format(vpool_name, self.api.wait_for_task(task)[0]))
             try:
                 self.vpools.append(VPoolHelper.get_vpool_by_name(vpool_name))
```

Comment:
Validate whether the vPool is running correctly. Perhaps the integration tests already have libraries to validate the sanity of vPools.
In current case the vPool may have been modelled, but failed in the middle of the creation process and this test will pass